assets/sass/all-digital/components/_card-group.scss

```diff
@@ -1,5 +1,5 @@
 .card-group {
-  margin-bottom: 24px;
+  margin-bottom: 12px;
```

Comment:
hmm this has changed recently, are we sure? they have weird rules where if its a card-group next to a card-group its 12, but if its card-group + heading or something it should be 24